pages/osx/sed.md

```diff
@@ -29,3 +29,7 @@
 - Replace separator / by any other character not used in the find or replace patterns, e.g., #:
 
 `sed 's#{{find}}#{{replace}}#' {{filename}}`
+
+- Remove line number 123 from a file, interactively with no backup:
```

Comment:
Do the command ask for confirmation? I want to be sure: Do this delete a line, or a number in the line?
```suggestion
- Remove a line from a file, prompting for confirmation:
```

Reply:
Hi @navarroaxel!
This command does NOT ask for confirmation and will delete the line matching that number. I've updated the PR, is this more clear?

Reply:
Resolved in a later convo